src/Tickets/Commerce/Order_Modifiers/Admin/Editor.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Editor class.
+ *
+ * @since TBD
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Tickets\Commerce\Order_Modifiers\Admin;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\StellarWP\Assets\Assets;
+use TEC\Tickets\Commerce\Order_Modifiers\API\Base_API;
+use TEC\Tickets\Commerce\Order_Modifiers\API\Fees;
+use TEC\Tickets\Commerce\Order_Modifiers\Traits\Asset_Build;
+use Tribe__Main as Common;
+use WP_Post;
+
+/**
+ * Class Editor
+ *
+ * @since TBD
+ */
+final class Editor extends Controller_Contract {
+
+	use Asset_Build;
+
+	/**
+	 * Registers the filters and actions hooks added by the controller.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	protected function do_register(): void {
+		/*
+		 * This is a workaround to avoid the following error:
+		 *
+		 * ReferenceError: Can't find variable: tec
+		 *
+		 * This error is caused by webpack trying to map the tec.tickets.orderModifiers.rest object
+		 * before is has been initialized in the browser. It's not yet known why this is happening.
+		 *
+		 * @todo: remove this workaround once the issue is fixed.
```

Comment:
I hadnt noticed that one. I am pretty sure its nothing. I removed it and its working fine. We are using very similar approach in seating without requiring a patch like that.

I think that @JPry misinterpreted another issue possibly. Jeremy could you elaborate please ?

Reply:
When I was first working on this portion of the feature, I kept getting an error in the console that the `tec.tickets.orderModifiers.rest` object didn't exist. As best I could tell, the `export` of that object was happening *before* it was actually defined. I wasn't able to figure out why that was the case, or how to solve it. This was my stop-gap solution to get it to function instead of wasting more time trying to figure it out.

If it works now without this, I'm glad to hear that. I would still like to know *why*, because I want to make sure I don't cause the same problem again.

I specifically left this comment to explain why it was here and so that we could remove this code if/when the issue I saw was resolved.